src/index.scss

```diff
@@ -26,3 +26,8 @@ main {
 .bx--content {
   padding: 0;
 }
+
+$feature-flags: (
+  ui-shell: true,
+  grid-columns-16: true,
+);
```

Comment:
Your padding in your preview is off and your repo is not full width: https://deploy-preview-2198--carbon-tutorial.netlify.com/#/repos

This block should come before the styles import `Note: Like before, the feature flag still needs to come before the Carbon styles.scss import.`

Reply:
done